docs/api-doc.md

```diff
@@ -1796,12 +1778,20 @@ Requires admin token
 ```bash
 Method: GET
 Type: 
-URL: {{server_url}}/securityQuestions/all
+URL: {{server_url}}/securityQuestions/:id
 ```
 
 
 
-### 6. Get SecurityQuestion
+***URL variables:***
+
+| Key | Value | Description |
+| --- | ------|-------------|
+| id | 1 |  |
+
+
+
+### 6. Update SecurityQuestion Copy
```

Comment:
users still can update a security question's active state to false while there is an instance of useranswer. is this intended behaviour?

Reply:
For now theres no restrictions on isActive as I'm not sure of the use cases except for the use case where the admin can only use the isActive is true questions

Reply:
ok